apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -316,6 +316,7 @@ const Ticket = new GQLListSchema('Ticket', {
                 delete: false,
             },
         },
+        //TODO(nas): Create an ability to override some fields for addressMeta field
```

Comment:
this fields has own logic and you don't need to override this fields by plugin

Reply:
Ok, I've deleted the comment.